plugin-ozone/pom.xml

```diff
@@ -28,6 +28,7 @@ limitations under the License.
     <name>Ozone Security Plugin</name>
     <description>Ozone Security Plugin</description>
     <properties>
+        <checkstyle.failOnViolation>true</checkstyle.failOnViolation>
```

Comment:
`<checkstyle.skip>false</checkstyle.skip>` would be needed as well.

Reply:
yup. added it.